### DISCUSSION TITLE
Just getting back into i3. Having a small monitor issue

### DISCUSSION THREAD

**Discussion by Phydoux65** [i3/i3#6163] [Q&A]
Hi all, I just came over from Reddit. Anyway,
I have a 3 monitor setup. I have a 55" TV above 2 smaller 22" monitors. So they're kind of in a pyramid scheme. 
So, the upper monitor (the 55" TV) is currently holding workspace 1. Then the monitor on the lower right is workspace 2 and the one on the lower left is workspace 3.
Again, It's not a big deal really. But is there any way to swap the 2 and 3 workspace monitors? 
I use xrandr and the monitors are setup appropriately (From the upper monitor, with the mouse, I can go down to the lower monitor on the right, then go left to the monitor on my left and then go back up to the upper monitor). So it's all setup physically correctly but for some reason, i3 has put workspace 2 on my right and workspace 3 on my left. A little strange bit again... Not a big deal if it's not fixable.

Thanks all!!!

**Comment by okraits** [comment]
This doesn't change the order of the monitors but you could specify that workspace 2 is always on the lower left and workspace 3 always on the lower right monitor:

https://i3wm.org/docs/userguide.html#workspace_screen

**Reply by Phydoux65** [reply]
So, all I had to do was go to the monitor with the 2 desktop on it and open up like 5, then I could go to the left monitor and make that one open tag 2. Now when I boot up, it starts up the way I want it. Thanks for your input. It's much appreciated!
